### PR TITLE
Report sarif warnings

### PR DESCRIPTION
**Pull request by ivy-cst**
This will make the build yellow until we have fixed all CVE's or ignored them.
Also the corresponding rebuild will be unstable: https://jenkins.ivyteam.io/job/core_rebuild-docker-image/
On 8.0 we have 41 warnings.
So someone should decide if we fix them or ignore them ...

### CODE REVIEW

**Review by ivy-lli — APPROVED** [review]
👍

File: docker-scout/Jenkinsfile [hunk 1]
```diff
@@ -81,6 +101,10 @@ def dockerScoutContainer() {
   return 'docker/scout-cli:1.4.1'
 }
 
+def axonivyEngineImageUrl(String version) {
+  return 'registry://axonivy/axonivy-engine:' + version
```
Comment (weissreto) [inline docker-scout/Jenkinsfile:105]: why you need registry:// here?

**Review by weissreto — APPROVED** [review]
As far as I understand it. Looks good

File: docker-scout/Jenkinsfile (hunk 1 above)
Comment (ivy-cst) [reply docker-scout/Jenkinsfile:105]: Then always the lates form the registry is used (not a cached one): https://github.com/docker/scout-cli/blob/main/docs/scout_cves.md#description

**Comment by alexsuter** [comment]
👍

**Pull request merged**